distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -53,138 +53,164 @@ The default metacard type has the following required attributes.
 Though the following attributes are required on all metacard types, setting their values is optional except for ID.
 
 .[[_metacard_required_attributes]]Metacard Required Attributes
-[cols="2m,2m,1,2", options="header"]
+[cols="2m,2m,1m,2,1", options="header"]
 |===
 
 |Metacard Constant
 |Attribute Name
 |Attribute Format
 |Description
+|Constraints
 
-|CONTENT_TYPE
-|metadata-content-type
-|STRING
-|Attribute name accessing for the metadata content type of a resource.
+|CHECKSUM
+|checksum
+|String
+|Checksum value for the primary resource for the metacard
+|< 1024 characters
 
-|CONTENT_TYPE_VERSION
-|metadata-content-type-version
-|STRING
-|Attribute name for the version of the metadata content accessing type of a resource.
+|CHECKSUM_ALGORITHM
+|checksum-algorithm
+|String
+|Algorithm used to calculate the checksum on the primary resource of the metacard.
+|< 1024 characters
 
 |CREATED
 |created
 |DATE
 |Attribute name for the date/time *this metacard* was created.
+|None
 
 |DATATYPE
 |datatype
 |String
 |The generic type of the resource.
+|Collection, Dataset, Event, Image, Interactive Resource, Service, Software, Sound, Text, Video, Document
 
-|DERIVED_RESOURCE_DOWNLOAD_URL
-|resource.derived-download-url
+|DESCRIPTION
+|description
 |String
-|Download URL for accessing the derived formats for the metacard resource.
+|An account of the resource.
+|None
+
+|EXPIRATION
+|expiration
+|DATE
+|The expiration date of the resource.
+|None
 
 |DERIVED_RESOURCE_URI
-|resource.derived-uri
+|resource-derived-uri
 |String
 |Location of the derived formats for the metacard resource.
+|Valid URI per RFC 2396
+
+|DERIVED_RESOURCE_DOWNLOAD_URL
+|resource-derived-download-url
+|String
+|Download URL for accessing the derived formats for the metacard resource.
+|Valid URL per RFC 2396
 
 |DESCRIPTION
 |description
 |String
 |An account of the resource.
-
-|EFFECTIVE
-|effective
-|DATE
-|Attribute name for the date/time of *the product* represented by the metacard.
+|None
 
 |EXPIRATION
 |expiration
 |DATE
-|Attribute name for the date/time the metacard is no longer valid and can be removed.
+|The expiration date of the resource.
+|None
 
 |ID
 |id
 |STRING
 |Attribute name for the ID of the metacard.
+|< 1024 characters
 
 |LANGUAGE
 |language
 |String
 |The language of the resource.
+|Alpha-3 language code per ISO_639-2
 
 |LOCATION
 |location
 |GEOMETRY
 |Attribute name for the location for this metacard.
+|Valid Well Known Text (WKT)
 
 |METACARD_CREATED
 |metacard.created
 |Date
 |The creation date of the metacard
+|None
 
 |METACARD_MODIFIED
 |metacard.modified
 |Date
 |The modified date of the metacard
+|None
 
 |METACARD_OWNER
 |metacard.owner
 |String
 |The email address of the metacard owner.
+|< 1024 characters
```

Comment:
I checked the metacardValidators and the attributeValidators and it looks like we don't validate against that. It's a single address, not sure if I should do more to note that beyond the singular "address" in the description.